frontend/src/tests/pages/HomePage.test.js

```diff
@@ -69,8 +69,6 @@ describe("HomePage tests", () => {
         expect(title).toBeInTheDocument();
         expect(typeof (title.textContent)).toBe('string');
         expect(title.textContent).toEqual('Howdy Farmer');
-
-        expect(() => screen.getAllByTestId(/commonsCard-button/)).toThrow('Unable to find an element');
```

Comment:
Is there a reason this assertion was deleted? This feature doesn't the homepage.

Reply:
Probably an accident while resolving merge conflicts.  I'll see what happens when we restore it.